test/libsolidity/semanticTests/arithmetics/signed_mod.sol

```diff
@@ -2,13 +2,21 @@ contract C {
     function f(int a, int b) public pure returns (int) {
         return a % b;
     }
+    function g(bool _check) public pure returns (int) {
+        int x = type(int).min;
+        if (_check) {
+            return x / -1;
+        } else unchecked {
```

Comment:
Do we want to allow this syntax, or should it only be possible to use `unchecked` in a statement inside a proper block?

Reply:
It's allowed with `try` and `if` so I don't see why not.